modules/notifications/assets/js/components/editor-v2.js

```diff
@@ -29,6 +29,16 @@ export const editorV2 = () => {
 				title: __( "What's New", 'elementor' ),
 				icon: () => <IconWithBadge invisible={ isRead } />,
 				onClick: () => {
+					elementor.editorEvents.dispatchEvent(
+						elementor.editorEvents.config.names.topBar.whatsNew,
```

Comment:
should we use some dynamic value to get a name here? 

or where is `documentSettings` is attached as example